doc/hardware/porting/board_porting.rst

```diff
@@ -34,39 +88,23 @@ You can visualize the hierarchy like this:
 Here are some examples. Notice how the SoC series and family levels are
 not always used.
 
-.. list-table::
-   :header-rows: 1
-
-   * - Board
-     - SoC
-     - SoC series
-     - SoC family
-     - CPU core
-     - Architecture
-   * - :ref:`nrf52dk_nrf52832 <nrf52dk_nrf52832>`
-     - nRF52832
-     - nRF52
-     - Nordic nRF5
-     - Arm Cortex-M4
-     - Arm
-   * - :ref:`frdm_k64f <frdm_k64f>`
-     - MK64F12
-     - Kinetis K6x
-     - NXP Kinetis
-     - Arm Cortex-M4
-     - Arm
-   * - :ref:`stm32h747i_disco <stm32h747i_disco_board>`
-     - STM32H747XI
-     - STM32H7
-     - STMicro STM32
-     - Arm Cortex-M7
-     - Arm
-   * - :ref:`rv32m1_vega_ri5cy <rv32m1_vega>`
-     - RV32M1
-     - (Not used)
-     - (Not used)
-     - RI5CY
-     - RISC-V
+.. table::
+
+   +--------------------------------------------------+-------------+-------------+---------------+---------------+----------------+--------------+
+   | Board                                            | Identifier  | SoC         | SoC Series    | SoC family    | CPU core       | Architecture |
+   +==================================================+=============+=============+===============+===============+================+==============+
+   | :ref:`nrf52dk <nrf52dk>`                         | /nRF52832   | nRF52832    | nRF52         | Nordic nRF5   | Arm Cortex-M4  | Arm          |
+   +--------------------------------------------------+-------------+-------------+---------------+---------------+----------------+--------------+
+   | :ref:`frdm_k64f <frdm_k64f>`                     |             | MK64F12     | Kinetis K6x   | NXP Kinetis   | Arm Cortex-M4  | Arm          |
+   +--------------------------------------------------+-------------+-------------+---------------+---------------+----------------+--------------+
+   | :ref:`stm32h747i_disco <stm32h747i_disco_board>` |             | STM32H747XI | STM32H7       | STMicro STM32 | Arm Cortex-M7  | Arm          |
```

Comment:
`stm32h747i_disco <stm32h747i_disco>`

Reply:
Note that this is ambiguous as STM32H747XI is a dual core SoC (M7/M4), so it depends what you want to illustrate here

Reply:
> `stm32h747i_disco <stm32h747i_disco>`

according to this line the `_board` is correct.
https://github.com/zephyrproject-rtos/zephyr/blob/a852186ace118de40329d76f3a5ce80aba81ef13/boards/boards_legacy/arm/stm32h747i_disco/doc/index.rst?plain=1#L1

but maybe there is a board porting PR which is updating this ?

> Note that this is ambiguous as STM32H747XI is a dual core SoC (M7/M4), so it depends what you want to illustrate here

Thanks for noticing.
I merely was updating the table to reflect the new format, so the basic purpose of the table haven't changed.
And while updating, I wanted to ensure that also a board with a multi-core SoC was also presented so that it was shown how that impacts the SoC, SoC series, and SoC family + architecture.

Guess I should extend `stm32h747i_disco` to show both cores (or remove it from the table, now that mimx8mp` presents similar).
A reason for having both mimx8mp and stm32h74i_disco could be to show both a dual core with same arch on both cores, and one with different archs.

If updating stm32h747i_disco in the table, then I prefer to have a fixed PR against collab-branch so I know final names / references to use.